h2o-algos/src/main/java/hex/tree/gbm/GBM.java

```diff
@@ -954,10 +954,6 @@ public HuberLeafMath(FrameMap frameMap, double huberDelta, Vec strata) {
 
     @Override
     public void map(Chunk[] cs) {
-      if (_strataMin < 0 || _strataMax < 0) {
-        Log.warn("No Huber math can be done since there's no strata.");
-        return;
-      }
```

Comment:
What is the reason to remove these lines of code? I am just curious. :)

Reply:
its does not make any sense, the strata can be negative, even when it is negative its still useful